README.rst

```diff
@@ -32,6 +32,17 @@ to support modern computational graph libraries such as PyTorch and
 TensorFlow in order to make use of features such as autodifferentiation
 and GPU acceleration.
 
+Try out now with Pyolite
+------------------------
+
+.. raw:: html
```

Comment:
Ah the use of a `raw` directive is a no go in the `README` as `twine` forbids it

```
Checking dist/pyhf-0.6.4.dev124-py3-none-any.whl: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 3[8](https://github.com/scikit-hep/pyhf/runs/5666625641?check_suite_focus=true#step:10:8): Warning: "raw" directive disabled.
Checking dist/pyhf-0.6.4.dev[12](https://github.com/scikit-hep/pyhf/runs/5666625641?check_suite_focus=true#step:10:12)4.tar.gz: FAILED
  `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 38: Warning: "raw" directive disabled.
```